data_pump/bitstreamformatregistry.py

```diff
@@ -0,0 +1,75 @@
+import logging
+
+from utils import read_json, convert_response_to_json, do_api_get_all, do_api_post
+
+
+def import_bitstreamformatregistry(bitstreamformat_id_dict,
+                                   unknown_format_id_val,
```

Comment:
parameter not used. Why is it necessary?

Reply:
we use it. we add there new value

Reply:
it is not used in this method

Reply:
I return this parameter